prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Brandon Moy';
+    console.log('value of fullName:', fullName);
+
+    var numberOfPets = 2;
+    console.log('value of numberOfPets:', numberOfPets);
+
+    var likesVegetables = 'true';
+    console.log('value of likesVegetables:', likesVegetables);
+
+    var nothing = 'null';
+    console.log('value of null:', nothing);
```

Comment:
I Brandon, for booleans and `null`, we leave out the quotations so that the javascript engine doesn't interpret them as string.

Reply:
Oops, updated the code, thank you!
![image](https://user-images.githubusercontent.com/109567499/183265501-2e2bfd48-de96-4fe9-b955-292ac837cc77.png)